src/main/java/nl/ordina/jobcrawler/scrapers/JobBirdScraper.java

```diff
@@ -36,36 +38,45 @@
 @Slf4j
 @Component
 public class JobBirdScraper extends VacancyScraper {
+    public void setLogService(LogService logService) {
+        this.logService = logService;
+    }
+
+    public void setDocumentService(DocumentService documentService) {
+        this.documentService = documentService;
+    }
 
+    private LogService logService = new LogService();
+    private DocumentService documentService  = new DocumentService();
 
-    private static final int MAX_NR_OF_PAGES = 25;  // 25 seems enough for demo purposes, can be up to approx 60
+    public static final int MAX_NR_OF_PAGES = 25;  // 25 seems enough for demo purposes, can be up to approx 60
```

Comment:
I searched the calls of this variable and they are all in the same class, which makes me wonder why this change was done

Reply:
no specific reason